libs/environment/src/components/network-loader/network-loader.tsx

```diff
@@ -28,6 +29,19 @@ export function NetworkLoader({
     return undefined;
   }, [VEGA_URL, cache]);
 
+  const offlineLayer = useMemo(() => {
```

Comment:
chore(blocking): I'd rather not block the UI if a user loses connection. Can we remove and address in a later sprint.

https://github.com/vegaprotocol/frontend-monorepo/issues/2922

Reply:
ok, agree